plugin-java/src/main/java/appland/javaAgent/AppMapJavaAgentDownloadService.java

```diff
@@ -102,6 +96,27 @@ public boolean downloadJavaAgentSync(@NotNull ProgressIndicator indicator) throw
         return true;
     }
 
+    @Nullable
+    private static Release.Asset getLatestAsset(@NotNull ProgressIndicator indicator) {
+        final Release[] releases = {getDefaultRelease(), GitHubRelease.INSTANCE};
```

Comment:
nit: Use `MavenRelease.INSTANCE` because the method mentions it's only for testing?